### PR TITLE
For #1640: adding author to wbs

### PR DESCRIPTION
**Pull request by paulodamaso**
For #1640:

- Added methods to Wbs
- Added test

### CODE REVIEW

**Comment by g4s8** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/zerocracy/farm/issues/2193#issuecomment-547775297))

@g4s8 Job [#2193](https://github.com/zerocracy/farm/issues/2193) is now in scope, role is `REV`

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
This pull request [#2193](https://github.com/zerocracy/farm/issues/2193) is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/159e8d13-eda3-4baa-b2da-28fd9a623419); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/com/zerocracy/pm/scope/WbsTest.java [hunk 1]
```diff
@@ -19,12 +19,19 @@
...
 import org.junit.Test;
 
 /**
  * Test case for {@link Wbs}.
  * @since 1.0
  * @checkstyle JavadocMethodCheck (500 lines)
+ * @todo #1640:30min Implement wbs.add(job, author) and wbs.author(job) methods to change
+ *  wbs.add() method behavior (every wbs item should have an author). After this, remove
+ *  addsAndRemovesJobs() test and replace it by addsAndRemovesJobWithAuthor() removing its ignore
+ *  annotation.
```
Comment (victornoel) [inline src/test/java/com/zerocracy/pm/scope/WbsTest.java:34]: @paulodamaso also the old `add(job)` method should be deleted from `Wbs`, no?

**Comment by victornoel** [comment]
@paulodamaso ping?

**Comment by victornoel** [comment]
@g4s8 should we refuse this PR?

**Comment by g4s8** [comment]
@paulodamaso can you please fix build problem?

**Comment by victornoel** [comment]
@g4s8 @paulodamaso  ping

**Comment by victornoel** [comment]
@paulodamaso I think you missed my comment there: https://github.com/zerocracy/farm/pull/2193#discussion_r409588160

**Comment by paulodamaso** [comment]
@victornoel Done, please take a look

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Code review was too long (20 days), architects (@g4s8) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by sereshqua** [comment]
@victornoel please make sure you will find at least 3 issues during next CR, thanks

**Comment by victornoel** [comment]
@sereshqua yep

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Pull request merged**